docs/traditional-media.md

```diff
@@ -211,6 +211,18 @@ pageClass: routes
 
 </Route>
 
+## iThome 台灣
+
+### Feeds
+
+<Route author="miles170" example="/ithometw/feeds/news" path="/ithometw/feeds/:category" :paramsDesc="['類別']" radar="1">
```

Comment:
Use path`/ithome/tw/feeds/:category`

Reply:
Should I merge `ithometw` into `ithome`?

Reply:
You can put it under tw folder.